src/main/java/com/orasi/bluesource/Employees.java

```diff
@@ -309,7 +312,100 @@ public boolean checkAccountPermissionOption(String strOption) {
 		catch (OptionNotInListboxException e){
 			return false;
 		}
-			
 	}
+	/**
+	 * Checks the manager dropdown for the employee name passed,
+	 * returns false when the name is not found and true when found
+	 * 
+	 * @param strOption the name of the employee you want to check for.
+	 * @return <code>true</code> if the name provided is available
+	 * in the dropdown, <code>false</code> otherwise.
+	 * @author Andrew McGrail
+	 */
+	
+	
+	public boolean checkManagerOption(String strOption) {
+		try{
+			lstManager.select(strOption);
+			return true;
+		}
+		catch (OptionNotInListboxException e){
+			return false;
+		}
+	}
+	
+	/**
+	 * This method returns the string of the success message
+	 * 
+	 * @return The success message in a String
+	 * @author Andrew McGrail
+	 */
+	public String getSuccessMessage() {
+		return driver.findElement(By.xpath("//*[@id='notification-area']/div")).getText().substring(2,35);
+	}
+	
+	/**
+	 * 
+	 * @return int - row number with the first non-Inactive User
+	 * @author Andrew McGrail
+	 */
+	public int findFirstActiveInTable(){
+		Integer intRow = null;
+		tblEmployees.syncVisible(2,true);
+		
+		//tblEmployees.getRowCount()
+		for (int i = 1; i < 50; i++) {
+			if (tblEmployees.getCellData(i, 12).equalsIgnoreCase("Permanent") | tblEmployees.getCellData(i, 12).equalsIgnoreCase("Contractor")) {
+				intRow = i;
+				break;
+			}
+		}
+		return intRow;
+	}
+	
+	/** 
+	 * This method selects the link of the first name for a given row
+	 * and column for the employee webtable
+	 * @param rowNum - The row number
+	 * @param columnNum - The column number
+	 * @author Andrew McGrail 
+	 */
+	public void selectEmployeeFirstname(int rowNum, int columnNum) {
```

Comment:
This looks like you are assuming in the test class you know what row the employee is in the table.  I don't remember what page this is in bluesource, but if it is a table of a bunch employees, then this could potentially fail, if the order of employees changes, or more employees are added, etc.  Isn't there a method like clickCellWithRowText?  Or could do the xpath dynamic by employee first name instead of the row/column num.

Reply:
This request didn't seem to register my update, for this issue I created another selectEmployee to use a passed first name, in case an employee was added or the table was changed after selecting the first non-Inactive employee.